Inutilização de faixa de numeração dos documentos fiscais

lfdivino opened the pull request.

lfdivino committed:
[ADD] Funcionalidade para inutilizar faixa de numeração

lfdivino committed:
[ADD] Visão e dados dos tipos de documentos inutilizados

lfdivino committed:
[FIX] Atributo de readonly dependendo do state da Inutilização

lfdivino committed:
[ADD] Verificar se existem faixa de números inutilizados

mileo commented on sped_nfe/data/sped.inutilizacao.tipo.documento.csv:
```diff
@@ -0,0 +1,34 @@
+"id","codigo","name","electronico"
```
Esta tabela atualmente se encontra no arquivo constante tributária, 

Veja se vc consegue utilizada-la ao invés desta

lfdivino committed:
[FIX] Tratar o resultado da chamada do super()

Tratar o resultado da chamada super() da _onchange_serie verificando
se foi retornado algum valor inicial, em caso negativo restornar
o resultado diretamente.
